src/Query/ResultPrinters/ResultPrinter.php

```diff
@@ -381,20 +381,20 @@ protected function handleNonFileResult( $result, QueryResult $results, $outputmo
 
 		// Apply intro parameter
 		if ( ( $this->mIntro ) && ( $results->getCount() > 0 ) ) {
-			if ( $outputmode == SMW_OUTPUT_HTML ) {
-				$result = Message::get( [ 'smw-parse', $this->mIntro ], Message::PARSE ) . $result;
-			} elseif ( $outputmode !== SMW_OUTPUT_RAW ) {
-				$result = $this->mIntro . $result;
-			}
+			if ( $outputmode == SMW_OUTPUT_HTML && $this->isHTML ) {
+ 				$result = Message::get( [ 'smw-parse', $this->mIntro ], Message::PARSE ) . $result;
+ 			} elseif ( $outputmode !== SMW_OUTPUT_RAW ) {
```

Comment:
We Somehow have some unrelated white space changes here (see left side, indicated by darker green).

Reply:
Weird, thanks, done.